packages/ds-healthcare-gov/src/components/Header/__snapshots__/Header.test.tsx.snap

```diff
@@ -24,7 +24,7 @@ exports[`Header renders Direct Enrollment banner 1`] = `
           href="/"
         >
           <span
-            class="hc-c-logo"
+            class="hc-c-logo "
```

Comment:
Would you be able to add a test to the `Header.test.tsx` file that provides those two new class props and checks that the corresponding rendered links have the class?

Reply:
Yep, will do 👍